frontend/config/firebase.js

```diff
@@ -0,0 +1,18 @@
+import firebase from 'firebase';
+const firebaseConfig = {
+  apiKey: process.env.NEXT_PUBLIC_APIKEY,
+  authDomain: process.env.NEXT_PUBLIC_AUTHDOMAIN,
+  projectId: process.env.NEXT_PUBLIC_PROJECTID,
+  storageBucket: process.env.NEXT_PUBLIC_STORAGEBUCKET,
+  messagingSenderId: process.env.NEXT_PUBLIC_MESSEGINGSENDID,
+  appId: process.env.NEXT_PUBLIC_APPID,
+};
+try {
+  firebase.initializeApp(firebaseConfig);
+} catch (err) {
+  if (!/already exists/.test(err.message)) {
+    console.error('Firebase initialization error', err.stack);
+  }
+}
+const fire = firebase;
+export default fire;
```

Comment:
แทนที่จะเป็น 
```
const fire = firebase;
export default fire;
```
ใช้เป็น
```
const storage = firebase.storage();
export { storage, firebase as default };
```
น่าจะดีกว่า

Reply:
อ้างอิงจากตรงนี้ (https://dev.to/itnext/how-to-do-image-upload-with-firebase-in-react-cpj)
![image](https://user-images.githubusercontent.com/46840981/108803977-a82a7680-75ce-11eb-840e-b68a106c3234.png)

Reply:
Note: เอาไว้ pr หน้าก็ได้